utils/lib/DataSource.ts

```diff
@@ -97,7 +97,7 @@ class DataSource extends Component<DataSourceConfig, string> {
       icon: this._getIconUrl(),
       install: this._parseInstall(),
       categoryTerms: categoryTerms && categoryTerms.map((t) => t.trim()),
-      keywords: keywords && keywords.map((k) => k.trim()),
+      keywords: keywords ? keywords.map((k) => k.trim()) : [],
```

Comment:
I know the ticket doesn't call this out, but can we also do this for quickstarts?
Also thinking `categoryTerms` should get the same treatment...

Reply:
@aswanson-nr it looks like quickstarts already have this behavior https://github.com/newrelic/newrelic-quickstarts/blob/269e82d447298281c9af044db1f64f5a738ecb87/utils/lib/Quickstart.ts#L176